web_calendar_year/static/src/js/web_calendar_year.js

```diff
@@ -0,0 +1,39 @@
+openerp.web_calendar_year = function(instance) {
+
+    instance.web_calendar.CalendarView = instance.web_calendar.CalendarView.extend({
```

Comment:
dont do this, this changes the reference of the object. Use `instance.web_calendar.CalendarView.include`